src/OpenApi/src/Transformers/TypeBasedOpenApiSchemaTransformer.cs

```diff
@@ -13,31 +13,36 @@ internal sealed class TypeBasedOpenApiSchemaTransformer : IOpenApiSchemaTransfor
     [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors)]
     private readonly Type _transformerType;
     private readonly ObjectFactory _transformerFactory;
+    private IOpenApiSchemaTransformer? _transformer;
```

Comment:
This is per invocation state, we can't store that on the class.

Reply:
See [Manage activated IOpenApiSchemaTransformer in OpenApiSchemaService](https://github.com/dotnet/aspnetcore/pull/56709/commits/96531784bba9bef958619467413998267bd16cad)